src/runtime/config.ts

```diff
@@ -15,24 +15,46 @@ const ENV_PREFIX_ALT =
 const _sharedRuntimeConfig = _deepFreeze(
   _applyEnv(klona(_inlineRuntimeConfig))
 );
+const getRuntimeConfig = () => ({
+  ..._sharedRuntimeConfig,
+  ...envToRuntimeObject(globalThis.__env__ ?? {}),
+});
+
 export function useRuntimeConfig<
   T extends NitroRuntimeConfig = NitroRuntimeConfig
 >(event?: H3Event): T {
   // Backwards compatibility with ambient context
   if (!event) {
-    return _sharedRuntimeConfig as T;
+    return getRuntimeConfig() as T;
   }
   // Reuse cached runtime config from event context
   if (event.context.nitro.runtimeConfig) {
     return event.context.nitro.runtimeConfig;
   }
   // Prepare runtime config for event context
-  const runtimeConfig = klona(_inlineRuntimeConfig) as T;
+  const runtimeConfig = klona(getRuntimeConfig()) as T;
   _applyEnv(runtimeConfig);
   event.context.nitro.runtimeConfig = runtimeConfig;
   return runtimeConfig;
 }
 
+export function envToRuntimeObject(
+  env: Record<string, unknown>,
+  prefixes = [ENV_PREFIX, "NUXT_", ENV_PREFIX_ALT]
```

Comment:
This shouldn't be hardcoded like this. `ENV_PREFIX_ALT` from nuxt is set to `NUXT_` already

Reply:
Are you saying this will work with Nuxt even if we remove "NUXT_" from this array?